Local dev docker instructions

ZachAttacksProblems opened the pull request:
The instructions are a bit long for part of a "quickstart".
Do you know what the RAM resource limit is? (I suggested 4GB but that might be very wrong). 
I believe the Disk limit is 1GB?

ZachAttacksProblems committed:
Added instructions for deploying docker image

byrcyb commented:
I checked with Ian and the RAM resource limit is 2GB. Disk limit is 1 GB but for most of the data science interns that's been bumped up to 2GB or 5GB but it's ok to stick with 1GB for disk.

ZachAttacksProblems committed:
Update quickstart.md

Removed confusing intermediate command.

ZachAttacksProblems committed:
Update quickstart with correct usage limits

Updated to include correct disk and memory limits for docker hub users.

ZachAttacksProblems committed:
Calrify docker command usage

More explicitly list which docker run flags must be present. 
Also indicated that some docker run flags can be omitted, but only if 
you are familiar with docker.

byrcyb reviewed (approved):
Tested and changes look good.
